astroid/nodes/as_string.py

```diff
@@ -14,22 +14,6 @@
 
 if TYPE_CHECKING:
     from astroid import objects
-    from astroid.nodes import Const
-    from astroid.nodes.node_classes import (
-        Interpolation,
-        Match,
-        MatchAs,
-        MatchCase,
-        MatchClass,
-        MatchMapping,
-        MatchOr,
-        MatchSequence,
-        MatchSingleton,
-        MatchStar,
-        MatchValue,
-        TemplateStr,
-        Unknown,
-    )
```

Comment:
Risky to do this in the astroid.nodes package as it's going to easily create circular import, no ?